lib/vericite/response.rb

```diff
@@ -0,0 +1,112 @@
+#
```

Comment:
This file needs to be revisited as well.  There are attr_accessors followed by get methods?  Some understanding of what is trying to be accomplished would be very helpful.